src/app/routes/registration/request-claim/request-claim.component.ts

```diff
@@ -666,10 +666,7 @@ export class RequestClaimComponent implements OnInit, SubjectElements {
             .map((role) => {
               this.selectedRole = role.definition;
               this.selectedNamespace = role.namespace;
-              this.fieldList =
-                this.selectedRole?.requestorFields ||
-                this.selectedRole?.fields ||
-                [];
+              this.fieldList = this.selectedRole?.requestorFields || [];
```

Comment:
@Harasz supporting `fields` is no needed anymore?

Reply:
We drop support for `fields` in the ICL